### PR TITLE
[ready-for-core-team-review]CRM-19585, added function to calculate net amount if contribution has…

### PR DESCRIPTION
**Pull request by pradpnayak**
… tax amount

added test for the same

----------------------------------------
* CRM-19585: Sales tax issue
  https://issues.civicrm.org/jira/browse/CRM-19585

### CODE REVIEW

**Comment by pradpnayak** [comment]
Added notice fix

**Comment by jitendrapurohit** [comment]
jenkins, retest this please

File: tests/phpunit/CRM/Contribute/BAO/ContributionTest.php [hunk 1]
```diff
@@ -938,6 +938,7 @@ public function testAllowUpdateRevenueRecognitionDate() {
   }
 
   /**
+<<<<<<< HEAD
```
Comment (jitendrapurohit) [inline tests/phpunit/CRM/Contribute/BAO/ContributionTest.php]: @pradpnayak rebase fix needed ?

**Comment by pradpnayak** [comment]
@jitendrapurohit  i have pushed fix.

**Comment by jitendrapurohit** [comment]
This works fine on `Edit Contribution` Form. Should this PR also include the net_amount displayed on the View Page? Currently `net_amount` is displayed as total + tax on View Contribution.

**Comment by pradpnayak** [comment]
I think on view page the net amount should be  total + tax (this is how its stored in db). On New/Edit form the tax is calculated on total amount after form is submitted (and its also displayed on form after calculating for total amount field )

**Review by jitendrapurohit — APPROVED** [review]

**Pull request merged**